sdf-schema.json

```diff
@@ -122,104 +127,138 @@
           },
           "minitems": 1
         },
-        "optional": {
-          "type": "boolean"
+        "required": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "minitems": 1
         }
-      }
+      },
+      "required": ["id"]
     },
+
     "thingqualities": {
-      "type": "object",
-      "properties": {
-        "odmObject": {"$ref": "#/properties/odmObject"},
-        "odmThing": {"$ref": "#/properties/odmThing"},
-        "odmView": {"$ref": "#/properties/odmView"}
-      },
-      "additionalProperties": {"$ref": "#/definitions/commonqualities"}
+      "allOf": [
```

Comment:
I am ok with using allOf in the schema, 
we should avoid having allOf in the ODM descriptions

Reply:
I haven't added "allOf" to the language, it's only used in the schema for the language, to call multiple schemas to validate an instance.